Eatery/Onboarding/Controllers/OnboardingViewController.swift

```diff
@@ -14,24 +14,21 @@ protocol OnboardingViewControllerDelegate {
 
 class OnboardingViewController: UIViewController {
 
-    let stackView = UIStackView()
-    let subtitleLabel = UILabel()
-    
+    private let stackView = UIStackView()
     private let titleLabel = UILabel()
-    private var imageView: UIImageView!
-    private let nextButton = UIButton()
+    private let subtitleLabel = UILabel()
 
     private var onboardingTitle: String!
     private var onboardingSubtitle: String!
-    private var onboardingImage: UIImage?
+
+    let contentView = UIView()
```

Comment:
Can this be private?

Reply:
Can't be since we are accessing it in OnboardingViewController's subclasses